extendr-api/src/na.rs

```diff
@@ -23,9 +23,16 @@ lazy_static! {
     };
 }
 
-/// Return true if this primitive is `NA`.
+/// Trait signifying if a given scalar type may be represented as an `NA` vlaue
+/// in an R vector.
+///
+/// The `is_na_scalar` method is intended to be used on individual
+/// elements of a vector. It is not vectorized and will return `false` if the
+/// vector has a length greater than 1.
 pub trait CanBeNA {
+    /// Returns `true` if it is `NA`.
     fn is_na(&self) -> bool;
```

Comment:
I see the `is_na_scalar()` method associated with the `Robj`  struct but not the `CanBeNA` trait. Should the Robj impl `CanBeNA` and not have its own method?